File: JS/parOuImpar.js
const assert = require('assert');

function parOuImpar(n) {
  return n % 2;
}

try {
  assert.equal(0,parOuImpar(2));
  assert.equal(1,parOuImpar(3));
} catch (e) {
  console.log(e);
}

File: C/parOuImpar.c
#include<stdio.h>
#include<assert.h>

int parOuImpar(int n) 
{
  return n % 2;
}

int main()
{
  assert(0 == parOuImpar(2));
  assert(1 == parOuImpar(3));
  return 0;
}